src/vapid-helper.js

```diff
@@ -90,6 +98,44 @@ function validatePrivateKey(privateKey) {
 }
 
 /**
+ * Given the number of seconds calculates
+ * the expiration in the future by adding the passed `numSeconds`
+ * with the current seconds from Unix Epoch
+ *
+ * @param {Number} numSeconds Number of seconds to be added
+ * @return {Number} Future expiration in seconds
+ */
+function getFutureExpirationTimestamp(numSeconds) {
+  const futureExp = new Date();
+  futureExp.setSeconds(futureExp.getSeconds() + numSeconds);
+  return Math.floor(futureExp.getTime() / 1000);
```

Comment:
@marco-c actually I should use setUTCSeconds and getUTCSeconds in the line 110 but It's still not clear to me if getUTCSeconds is what we need here, we need futureExpirationTimestamp which is calculated by adding the current seconds in UNIX epoch with the `numSeconds` 

getUTCSeconds is going to return a value between 0 and 59. Does that clear it out?